ban/commands/ignsna.py

```diff
@@ -0,0 +1,48 @@
+import os
+from ban.commands import command, report
+from ban.core.models import (Municipality, PostCode)
+from .helpers import session, batch, nodiff
+
+__namespace__ = 'import'
+
+
+@command
+@nodiff
+def ignsna(path, **kwargs):
+    """Import from IGN/Laposte BDUNI
+    :param path: directory location of the files hsp7aaaa.ai, hsv7aaaa.ai and hsw4aaaa.ai"""
+
+    municipality_zipcode_file = os.path.join(path, 'hsp7aaaa.ai')
+    if municipality_zipcode_file is not None:
+        lines = open(municipality_zipcode_file).readlines()
```

Comment:
doing so, you don't close the file ;)
Not a big deal on a one-time command, but better to do it anyway.
```python
with open(xxx) as f:
    lines = f.readlines()
```

But actually, you don't need to call readlines, you can just pass the file to batch:
```python
with open(xxx) as f:
    batch(func, f…)
```